doc/classes/Array.xml

```diff
@@ -317,19 +340,19 @@
 		<method name="get_typed_builtin" qualifiers="const">
 			<return type="int" />
 			<description>
-				Returns the [enum Variant.Type] constant for a typed array. If the [Array] is not typed, returns [constant TYPE_NIL].
+				Returns the built-in type of the typed array as a [enum Variant.Type] constant. If the array is not typed, returns [constant TYPE_NIL].
 			</description>
 		</method>
 		<method name="get_typed_class_name" qualifiers="const">
 			<return type="StringName" />
 			<description>
-				Returns a class name of a typed [Array] of type [constant TYPE_OBJECT].
+				Returns the [b]native[/b] class name of the typed array if the built-in type is [constant TYPE_OBJECT]. Otherwise, this method returns an empty string.
 			</description>
 		</method>
 		<method name="get_typed_script" qualifiers="const">
 			<return type="Variant" />
 			<description>
-				Returns the script associated with a typed array tied to a class name.
+				Returns the script associated with the typed array. This method returns a [Script] instance or [code]null[/code].
```

Comment:
Actually it returns `Object::null`, but I think it's a bug.

```gdscript
print(null) # <null>
print([].get_typed_script()) # <Object#null>
```

Reply:
Now I'm not sure if this is a bug, this is the behavior of all `Ref<T>` properties and method return values. Should we draw the reader's attention to this difference?

```gdscript
var sprite := Sprite2D.new()
print(sprite.texture) # <Object#null>
print(sprite.texture == null) # true
print(is_same(sprite.texture, null)) # false
print(typeof(sprite.texture)) # 24
print(typeof(null)) # 0
```